neqo-common/src/log.rs

```diff
@@ -50,7 +50,7 @@ fn since_start() -> Duration {
     START_TIME.get_or_init(Instant::now).elapsed()
 }
 
-pub fn init(level_filter: Option<log::LevelFilter>) {
+pub fn init(level_filter: Option<::log::LevelFilter>) {
```

Comment:
Unrelated change?

Reply:
Thanks. Addressed in https://github.com/mozilla/neqo/pull/1920/commits/c7b446de1be992a8d1b85f029386231e2a8926f3.